src/components/icon-card-link-grid-list/index.tsx

```diff
@@ -4,19 +4,19 @@ import { IconCardLinkGridListProps, IconCard } from './types'
 
 function IconCardLinkGridList({
   cards,
-  productSlug,
+  productSlug = 'hcp',
```

Comment:
This isn't quite what I was suggesting. This is now optional which is great, but I think this default value should be placed in `IconCardLink`'s `productSlug` property instead (has a wider impact). That would mean also making `IconCardLink`'s `productSlug` optional, and then the `IconCard` interface could be updated to this:

```tsx
export type IconCard = Pick<IconCardLinkProps, 'icon' | 'productSlug' | 'text' | 'url'>
```

and then `IconCardLinkGridList`'s would be effectively optional as well. Does that make sense?

Reply:
yep, makes snese! ✅ have moved the optional prop down a level